person.rb

```diff
@@ -10,6 +10,7 @@ def initialize(age, name = 'unknown', parent_permission: true)
     @age = age
     @parent_permission = parent_permission
     super(name)
+    @rentals = []
   end
 
   def can_use_services?
```

Comment:
- [ ] As stated in the project requirement A person has many rentals: you should be implemented **A method for adding rentals, T**he person itself should be sent as a parameter to create the new rental achieving the 'has-many' association Just like this :-

```
def add_rental(book, date)
    Rental.new(date, book, self)
  end
```